CHANGELOG.md

```diff
@@ -20,7 +20,13 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 + Verify some constants / equations and remove the comments questioning them
 + Relocate the time resolution of wmodels to one spot
 + Skip test_scenario_3 until a new golden dataset is created
-+ Update environment with scipy version minimum and requiring pybind11
++ Update environment with scipy version minimum and requiring pybind11 (the latter for Apple ARM64 with rosetta2)
```

Comment:
In a previous PR - Jeremy wanted to specify why pybind11 was required. This minor comment in the changelog was added.